src/service/fixed-layer.js

```diff
@@ -162,6 +234,19 @@ export class FixedLayer {
           ' as Google\'s because the fixed or sticky positioning might have' +
           ' slightly different layout.');
     }
+  }
+
+  /**
+   * @param {!Node} node
+   * @param {boolean=} opt_lightboxMode
+   * @private
+   */
+  scanNode_(node, opt_lightboxMode) {
+    this.trySetupSelectorsNoInline(node, devAssert(this.fixedSelectors_),
+        devAssert(this.stickySelectors_), opt_lightboxMode);
```

Comment:
Also, there's no need to pass around parameters if they're just instance properties.

Reply:
Tried to preserve the functional style but sure.